css/style.css

```diff
@@ -235,3 +231,17 @@ header {
     color: var(--var-white-color);
     background: var(--var-violet-color-v3);
 }
+
+.load-more-button {
+    display: block;
```

Comment:
`display` is by default `block` for most elements if not `inline` for span. Are we resetting `display` here? If not we can remove it as it is redundant.

Reply:
@hhu-wtag bhai the button is inline-block by default. I need to center the button so i made its display block and set margin auto to left and right.